packages/foreman/foreman/foreman.service

```diff
@@ -5,11 +5,11 @@ After=network.target remote-fs.target nss-lookup.target
 
 [Service]
 Type=simple
-User=foreman
+User=$FOREMAN_USER
 TimeoutSec=300
 WorkingDirectory=/usr/share/foreman
-ExecStart=/usr/bin/scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT"
-Environment=FOREMAN_ENV=production FOREMAN_PORT=3000
+ExecStart=/usr/bin/scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT -b $FOREMAN_IFACE"
```

Comment:
This is based on https://github.com/theforeman/puppet-foreman/blob/7dfb21677cd1dfe98e397527de2e18b19f8dcf38/templates/foreman.default.erb#L10 but it looks like the installer doesn't set it. I'd be open to to naming it something more correct, like `FOREMAN_BIND`

Reply:
ACK - I like FOREMAN_BIND much better.